Quentinchampenois/decidim-module-user_exporter
# Feature export users

### Description 

Allow administrator to export user data using DataPortability.

### How to test

Generate a development app : `bundle exec rake development_app`
Run application
Log as admin
Go to Participants > Participants
Export data 
Check at the letter opener

### Tasks

- [x] Export using Job
